### PR TITLE
coafile.rst:Add slashes

### PR DESCRIPTION
**Pull request by ManthanKeim**
This commit ensures that people recognize backslashes and forward slashes immediately.
For eg.
Forward slash - /
Backslash - \
Double Backslash - \\

Closes https://github.com/coala/documentation/issues/571

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [ ] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [ ] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `corobo mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

File: Users/coafile.rst [hunk 1]
```diff
@@ -162,9 +162,9 @@ Similarly, you can also set a value to multiple keys:
 ``key_1, key_2 = value`` is equivalent to ``key_1 = value`` and
 ``key_2 = value`` in separate lines.
 
-As the backslash is the escape character it is recommended to use
-forward slashes as path separator even on Windows (to keep relative
-paths platform independent), use double-backslashes if you really mean a
+As the backslash :math:`"\"` is the escape character it is recommended to use
```
Comment (li-boxuan) [inline Users/coafile.rst]: Remove the space at the end of this line.

File: Users/coafile.rst [hunk 2]
```diff
@@ -162,9 +162,9 @@ Similarly, you can also set a value to multiple keys:
 ``key_1, key_2 = value`` is equivalent to ``key_1 = value`` and
 ``key_2 = value`` in separate lines.
 
-As the backslash is the escape character it is recommended to use
-forward slashes as path separator even on Windows (to keep relative
-paths platform independent), use double-backslashes if you really mean a
+As the backslash :math:`"\"` is the escape character it is recommended to use 
+forward slashes :math:`"/"` as path separator even on Windows (to keep relative
```
Comment (li-boxuan) [inline Users/coafile.rst]: You can simply use ` ``/`` ` instead of ``` :math:`/` ```.
IMO ``` (``\``) ``` is better.

**Comment by li-boxuan** [comment]
Also please check your commit message. Your commit message has several redundant spaces.

**Comment by ManthanKeim** [comment]
@li-boxuan please check if the issue needs any more changes?

**Comment by ManthanKeim** [comment]
@li-boxuan the commits are now squashed. Please check.

**Comment by ManthanKeim** [comment]
@li-boxuan  I've done the said changes.

**Comment by jayvdb** [comment]
You need to use `git` to make the changes which @li-boxuan has requested.

read https://coala.io/newcomer , https://coala.io/git , etc again.

**Comment by jayvdb** [comment]
Still doesnt comform with https://coala.io/commit .

**Comment by ManthanKeim** [comment]
> Still doesnt comform with https://coala.io/commit .

is space redundancy the issue?

**Comment by PrajwalM2212** [comment]
*This commit ensures that people recognizes backslashes and  forward slashes immediately.*
There are two spaces between **backslashes and  forward**

**Comment by jayvdb** [comment]
ack https://github.com/coala/documentation/pull/572/commits/b169b848a3c367789bcc39308c28fcf7892d3126

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Pull request merged**